examples/handlebars.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Image with handlebars template</title>
+    <style>
+      html,
+      body {
+        height: 1080px;
+        width: 1920px;
+      }
+      body {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        font-size: 5vh;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>Hello {{you}}</h1>
```

Comment:
I would prefer `name` rather than `you` with `"name": "you"` in the `handlebars.json` file. What do you think of it?

Reply:
As you wish